### DISCUSSION TITLE
Got a Feature Request? Make it more appealing with this one simple trick!

### DISCUSSION THREAD

**Discussion by RoboMWM** [GriefPrevention/GriefPrevention#1137] [Ideas]
Want to beef up your feature/addon request and make it more easily understood? Copy this template and paste it into your ideas discussion thread:

```
**As** a player
**I want** to capture animals in a snowball
**Because** I wanna be the very best, Like no one ever was.

Optional questions:

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.
```

More tips on how to increase the chances of your feature/addon requests being implemented: https://web.archive.org/web/20240815053631/https://tanzu.vmware.com/content/blog/how-to-write-well-formed-user-stories

Requested features are almost always perfect for addons - so feel free to suggest this to others who may be interested in making an addon for GriefPrevention!

**Comment by Jikoo** [comment]
While I don't have the drive and whatnot right now, it would be super possible to make an action that attempts to match the format, creating a comment suggesting it for any issue tagged as an addon suggestion that doesn't meet spec.

**Reply by RoboMWM** [reply]
I haven't checked, but as long as actions don't have unnecessary permissions like permission to write to repo, I'm fine with them. I've otherwise avoided such bots due to this.

**Reply by Jikoo** [reply]
Well, it would need access to the repo's `GITHUB_TOKEN` secret to be able to post. If you're very hard against external actions like https://github.com/marketplace/actions/create-or-update-comment it would be possible to implement entirely manually.

**Reply by Jikoo** [reply]
You could, of course, also set up a separate bot user for security and not grant it any perms. Then you could use that user's PAT to post instead.

**Reply by RoboMWM** [reply]
ya, I haven't fully setup release drafter yet, but if it uses a personal access token then I'm ok with that since iirc you can control how much access that token has. And afaik all github actions allow use of specific versions to prevent unexpected changes in behavior with new updates.

**Comment by NotEvenAUnicorn** [comment]
It'd be awesome if there was a /trust node that allowed players to interact with entities in their claim, such as boats. Would this be possible in a future update??

**Comment by AldrianEmrys** [comment]
### Claim Chat Threads?

Would a separate with-in claim chat function be out of the blue to ask? I feel like a lot of players would enjoy this. I know a few would as we were just talking about it! Love your updates so far, thanks for keeping it going for so long.
